library.sh

```diff
@@ -794,11 +794,17 @@ function shellcheck_new_files() {
 }
 
 function latest_version() {
-  local semver=$(git describe --match "v[0-9]*" --abbrev=0)
-  local major_minor=$(echo "$semver" | cut -d. -f1-2)
+  if [ $(current_branch) = "master" ] || [ $(current_branch) = "main" ]; then
+    # For main branch, simply use git tag without major version, this will work even
+    # if the release tag is not in the main
+    git tag | sort -r --version-sort | head -n1
+  else
```

Comment:
I think this breaks how the function is used.

Reply:
It is fixing a problem that we are having in master/main branch. Other branches will stay the same.